redisinsight/ui/src/pages/rdi/instance-list/RdiInstancesListWrapper.tsx

```diff
@@ -63,6 +69,15 @@ const RdiInstancesListWrapper = ({ width, onEditInstance, editedInstance, onDele
     closePopover()
   }, [width])
 
+  const handleConnect = (id: string) => {
+    // TODO: update connect function (check connection first?)
+    // TODO: move reset browser context to instance rdi page
```

Comment:
Are these `TODO`s actionable?

Reply:
it will be covered in the another story - need to change routing a bit and navigation